docs/standard/components.md

```diff
@@ -14,37 +14,36 @@ ms.assetid: 2e38e9d9-8284-46ee-a15f-199adc4f26f4
+* Optionally, one or more application frameworks. Examples: ASP.NET, Windows Forms, and WPF are included in the .NET Framework.
+* Optionally, development tools. Some development tools are shared among multiple implementations.
 
-![All .NET Architectural Components Together](media/components.png)
-
-What follows is a brief explanation of each of the key components shown above.  
+What follows is a brief explanation of the key components mentioned in this overview.  
 
 ## .NET Standard
 
-The .NET Standard is a set of APIs which are implemented by a .NET runtime.
-
-More formally, it is a specification of .NET APIs which make up a uniform set of contracts that you compile your code against.  These contracts have underlying implementations for each .NET runtime.  This enables portability across different .NET runtimes, making it so that your code can effectively "run everywhere".
+The .NET Standard is a set of APIs which are implemented by the Base Class Library of a .NET implementation. More formally, it is a specification of .NET APIs which make up a uniform set of contracts that you compile your code against.  These contracts are implemented in each .NET implementation.  This enables portability across different .NET implementations, making it so that your code can effectively "run everywhere".
 
-The .NET Standard is also a build target. You can currently target .NET Standard 1.0-2.0. If your code targets a version of the .NET Standard, it is guaranteed to run on any .NET runtime which implements that version.
+The .NET Standard is also a target framework. You can currently target .NET Standard 1.0-2.0. If your code targets a version of the .NET Standard, it is guaranteed to run on any .NET implementation which supports that version of .NET Standard.
 
 To learn more about the .NET Standard and how to target it, see the [.NET Standard](net-standard.md) topic.
 
-## .NET runtimes
+## .NET implementations
 
-There are 3 primary .NET runtimes which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
+There are 3 primary .NET implementations which Microsoft actively develops and maintains: .NET Core, .NET Framework, and Mono for Xamarin.
 
 ### .NET Core
 
-.NET Core is a cross-platform runtime optimized for server workloads.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on .NET Core.  It is the runtime used by ASP.NET Core and the Universal Windows Platform (UWP).  It is modern, efficient, and designed to handle server and cloud workloads at scale.
+.NET Core is a cross-platform runtime optimized for server workloads.  It implements the .NET Standard, which means that any code that targets the .NET Standard can run on .NET Core.  It is the .NET implementation used by ASP.NET Core and the Universal Windows Platform (UWP).  It is modern, efficient, and designed to handle server and cloud workloads at scale.
```

Comment:
The constructions that use an "it" pronoun can be a bit slower to pick up by the reader. This is a really fine example imo ...

Is the "It" of the 2nd sentence referring to ".NET Core" or to the "runtime"?

... then I see a lot of "it"s in this para. I recommend refactoring to get rid of at least a few of them.

Contraction: It is :point_right: It's (occurs twice in the para)

.NET Core isn't just a "cross-platform runtime" as the opening sentence states. I suggest "includes" in there ...

> .NET Core includes a cross-platform runtime optimized for server workloads.

> ... any code that targets the .NET Standard can run on .NET Core ...

"any code" ... I hope so, but I have a bad feel'in about this, and I guess it's not good if it were to say "most code." Idk if I'm right/wrong about it today, but I'm just worried that ye 'olde `NotImplementedException` will pop up from time-to-time, so the statement is more of a goal than a hardcore fact if that's still the case.

Two-spaces between sentences here.